kotest-extensions/src/jvmMain/kotlin/io/kotest/extensions/time/ConstantNowExtensions.kt

```diff
@@ -105,6 +106,30 @@ class ConstantNowTestListener<Time : Temporal>(now: Time) :
    override suspend fun afterTest(testCase: TestCase, result: TestResult) {
       resetNow()
    }
+
+   override suspend fun beforeContainer(testCase: TestCase) {
+      if (testCase.type == TestType.Container) changeNow()
```

Comment:
We probably only need to call changenow resetnow on the beforeAny / beforeTest cases as that will cover it for every test.

Reply:
Thanks for pointing this out. I've mistakenly thought about the extension listeners differently and somehow thought that the functions executed are dependent on what the user writes in their tests, but that is obviously not the case 🙂

Reply:
Addressed in [`2ed277b` (#1589)](https://github.com/kotest/kotest/pull/1589/commits/2ed277b0b70c6e9e285e8abfca44e6c18aff7715) & [`9b20d34` (#1589)](https://github.com/kotest/kotest/pull/1589/commits/9b20d342b938d45446fd4b8fa237793e2317bb50)